From a2e79d4b0eaa577614c3a2395da25f120bba8bb0 Mon Sep 17 00:00:00 2001
From: Wolfgang Grandegger <wg@aries-embedded.de>
Date: Wed, 23 Oct 2019 15:24:53 +0200
Subject: [PATCH 4/6] Fix bug with "use a unique name to register freeze
 controller"

The commit 93bfe5a933d5f9fa904cf9806e0bdd7e3465c06e did introduce
a bug not taking into account a recent API change.

Signed-off-by: Wolfgang Grandegger <wg@aries-embedded.de>
---
 drivers/fpga/altera-freeze-bridge.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/fpga/altera-freeze-bridge.c b/drivers/fpga/altera-freeze-bridge.c
index bc986db..8c2a624 100644
--- a/drivers/fpga/altera-freeze-bridge.c
+++ b/drivers/fpga/altera-freeze-bridge.c
@@ -243,7 +243,7 @@ static int altera_freeze_br_probe(struct platform_device *pdev)
 
 	priv->base_addr = base_addr;
 
-	return fpga_bridge_register(dev, dev_name(dev),
+	br = fpga_bridge_create(dev, dev_name(dev),
 				&altera_freeze_br_br_ops, priv);
 	if (!br)
 		return -ENOMEM;
-- 
1.9.1

